js/main.js

```diff
@@ -1,70 +1,90 @@
+var player;
```

Comment:
Please use an indentation of 2 or 4 and be consistent in all `js` files.

Reply:
@michael-riha This was meant to apply for the whole file not only for this line 😄 

This line was ok to be at 0 ... but some other nested parts in the JS code are indented in a quite chaotic way.

Reply:
Have to switch from WebOS IDE to other editor, thanks!

Reply:
DONE, hopefully